portal-web/test/functional/com/liferay/portalweb/macros/Audit.macro

```diff
@@ -7,7 +7,7 @@ definition {
 
 		ClickNoError(
 			key_menuItem = "Export Audit Events",
-			locator1 = "MenuItem#ANY_MENU_ITEM");
+			locator1 = "MenuItem#ANY_MENU_ITEM_1");
```

Comment:
@melodywu Could you define a more specific name?

Reply:
Ok and done